card/spec/lib/card/set/trait_spec.rb

```diff
@@ -0,0 +1,62 @@
+
+describe Card::Set::Trait do
+  class Card
+    module Set
+      module Type
+        module Phrase
+          extend Card::Set
+          card_accessor :write, type: :pointer
+          card_accessor :read, type_id: Card::PointerID
```

Comment:
`card_accessor :write, type_id: :pointer` creates a basic card but at least `left` is working. To avoid a misunderstanding: this is not about getting that card_accessor working it's about using the `type` argument. We should either support that or at least give a sensible error message if somebody uses `type` instead of `type_id` (because we support it almost everywhere else). It took me a whole day to figure out that that was the reason while scored formulas were not working on wikirate.